src/app/features/wallets/transfer/transfer.page.ts

```diff
@@ -77,38 +64,23 @@ export class TransferPage {
   ionViewWillEnter() {
     this.blockingActionService
       .run$(this.diaBackendWalletService.syncIntegrityAndAssetWalletBalance$())
-      .pipe(untilDestroyed(this))
+      .pipe(
+        catchError(() =>
+          this.errorService.toastError$(
+            this.translocoService.translate(
+              `error.wallets.cannotRefreshBalance`
+            )
+          )
+        ),
+        untilDestroyed(this)
+      )
       .subscribe();
   }
 
   onInputTransferAmount() {
     this.readyToSend = false;
   }
 
-  hasSufficientBalance$() {
-    return combineLatest([
-      this.assetWalletBscNumBalance$,
-      this.integrityWalletBscNumBalance$,
-      this.totalCost$,
-    ]).pipe(
-      first(),
-      concatMap(
-        ([assetWalletBscNumBalance, integrityWalletBscNum, totalCost]) => {
-          if (
-            (this.mode === 'withdraw' &&
-              totalCost > assetWalletBscNumBalance) ||
-            (this.mode === 'deposit' && totalCost > integrityWalletBscNum)
-          ) {
-            throw new Error(
-              this.translocoService.translate('error.wallets.insufficientFund')
-            );
-          }
-          return of(true);
-        }
-      )
-    );
-  }
-
```

Comment:
Remove balance checking logic since backend is already checking it when create order